components/index/Calendar.tsx

```diff
@@ -78,9 +78,13 @@ const Calendar = ({
     useEffect(() => {
       //지금 로그인한 loggedId(memeberId) 구하기 -> 상위 index 컴포넌트에서 받아옴
       const getRecivedPresentList = async () => {
-        if (loggedId !== null) {
-          const res = await setGetNumberOfReceivedPresents(loggedId);
-          setReceivePresentList(await res.data.data);
+        if (loggedId !== undefined && loggedId !== null) {
```

Comment:
```suggestion
        if (!loggedId) {
```
요렇게 나타낼 수도 있을 것 같아요!